bioconda_cli/util.py

```diff
@@ -90,8 +90,14 @@ def get_package_binaries(package, version) -> List[pathlib.Path]:
     # The binaries in a given package are listed in the files key of the metadata file
     with metadata[0].open() as fp:
         parsed = json.load(fp)
-        # Only return binaries, not just any package file. Their actual location is relative to the prefix
-        return [env_path / f for f in parsed["files"] if f.startswith("bin/")]
+        # Only return binaries, not just any package file. 
+        # Their actual location is relative to the prefix
+        # Filter out .PACKAGENAME-post-link.sh and .PACKAGENAME-pre-unlink.sh
+        return [env_path / f for f in parsed["files"] if 
```

Comment:
Rather than recalculate these skippable binaries each iteration, could you put them into a `set` and check that the filepath is not in that set? Also please use `format()` rather than the `%` operator.

Reply:
kike so?

Reply:
Yep, exactly